### PR TITLE
Surport Swift 5

### PR DESCRIPTION
**Pull request by ceeyang**
## Checklist

- [x] New Extension
- [x] New Test
- [x] Changed more than one extension, but all changes are related
- [ ] Trivial change (doesn't require changelog)

### CODE REVIEW

File: Sources/DateExtensions.swift [hunk 1]
```diff
@@ -137,29 +137,29 @@ extension Date {
...
-        } else if components.minute! >= 1 {
-            return TimePassed.minute(components.minute!)
-        } else if components.second! >= 1 {
-            return TimePassed.second(components.second!)
-        } else {
-            return TimePassed.now
-        }
-    }
-    
+//    public func timePassed() -> TimePassed {
```
Comment (Khalian) [inline Sources/DateExtensions.swift]: Why are we commenting this out?

File: Sources/IntExtensions.swift [hunk 2]
```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import AVFoundation
```
Comment (Khalian) [inline Sources/IntExtensions.swift]: Why do we need this ?

File: Sources/NSObjectExtensions.swift [hunk 3]
```diff
@@ -6,6 +6,8 @@
 //  Copyright (c) 2015 Goktug Yilmaz. All rights reserved.
 //
 
+import Foundation
```
Comment (Khalian) [inline Sources/NSObjectExtensions.swift]: Do we need this ?

**Comment by Khalian** [comment]
I have created a remote branch ceeyang-Swift5.0, I will keep pushing in changes to fix builds and stuff, but I would need your help since I am out of date on all the XCode versions and MacOSX versions for latest swift stuff.

**Comment by ceeyang** [comment]
I will make a new PL

**Pull request closed without merging**